src/themes/theme.ts

```diff
@@ -0,0 +1,64 @@
+export interface Palette {
```

Comment:
This is the place where the type overrides need to happen - palettes, etc.

We should deprecate our theme-type-definitions and integrate/merge them with the material ones.

Once that is done most of the Theme imports and specified like below will become redundant.
From:
```jsx
import { Theme } from '../themes';

<Card
    classes={{
      background: (theme: Theme) => theme.palette.my_special_color,
    }}
/>
```
To:
```jsx
import { Theme } from '@material-ui/core';
<Card
    classes={{
      background: (theme) => theme.palette.my_special_color,
    }}
/>
```
We can close the review without changing the entirety of the Theme imports to material-ui, but the Theme definitions needs to be merged.